README.md

```diff
@@ -33,6 +33,7 @@ Source code located under `/server`. This Django API provides data such as a lis
 * Set up the Django settings environment variable: `export DJANGO_SETTINGS_MODULE=lostandfound.settings`. To avoid having to do this every time you can add this line to the `activate` script file located in your virtual environment `env/bin/activate`
 * Activate the environment: `source env/bin/activate`
 * Install dependencies: `pip install -r requirements.txt`
+* Make migrations to the database python manage.py migrate
```

Comment:
Also include `python manage.py createsuperuser`, because this is only how they will have access to the admin backend. The command should  be after `line 36` and before `line 37`